definitions/features/redis.rb

```diff
@@ -20,11 +20,19 @@ class << self
     SCL_NAME = 'rh-redis5'.freeze
 
     def etc_prefix
-      "/etc/opt/rh/#{SCL_NAME}"
+      if el7?
+        "/etc/opt/rh/#{SCL_NAME}"
+      else
+        '/etc'
+      end
     end
 
     def scl_prefix
-      "#{SCL_NAME}-"
+      if el7?
+        "#{SCL_NAME}-"
+      else
```

Comment:
Do we require else block? because the string interpolation with `nil` should work?
~~~
2.6.6 :001 > a = nil
 => nil 
2.6.6 :003 > puts "#{a}amit"
amit
 => nil 
~~~

Reply:
Probably not, was just thinking of being explicit.